enews-node/app.js

```diff
@@ -0,0 +1,19 @@
+var NewsFetcher = require('./newsFetcher.js')
+
+var token = process.env.token;
+var newsFetcher = new NewsFetcher(token);
```

Comment:
We get it for free, if the env isn't set node throws an error
On 8 Feb 2016 8:45 a.m., "Paul Blundell" <notifications@github.com> wrote:

> In enews-node/app.js
> <https://github.com/novoda/spikes/pull/19#discussion_r52142936>:
>
> > @@ -0,0 +1,19 @@
> > +var NewsFetcher = require('./newsFetcher.js')
> > +
> > +var token = process.env.token;
> > +var newsFetcher = new NewsFetcher(token);
>
> you might want to do a sanity check that the process.env.token value has
> been set, so that if someone runes this script it's obvious what went
> wrong. Fail Fast
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/novoda/spikes/pull/19/files#r52142936>.
>